src/consensus/params.h

```diff
@@ -59,6 +58,8 @@ struct Params {
     int BIP66Height;
     /** Block height at which CSV (BIP68, BIP112 and BIP113) becomes active */
     int CSVHeight;
+    /** Block height at which Segwit (BIP141, BIP143 and BIP147) becomes active */
```

Comment:
I wonder if it's worth adding a comment here to explain that segwit v0 script rules are enforced on all blocks, other than the BIP 16 exception?